build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
     implementation "com.sun.xml.bind:jaxb-core:2.3.0.1"
     implementation "com.sun.xml.bind:jaxb-impl:2.3.3"
     implementation "javax.activation:activation:1.1.1"
+    implementation 'com.amazonaws:aws-java-sdk-sns:1.11.692'
```

Comment:
Is there an API we can use instead of integrating a library?

Reply:
Thanks for responding @tananaev.
It would be better if we use SDK instead of using bare API itself. As we underlying authentication/authorisation and other complexities are taken care by the SDK itself. Also, we wont have to deal with the security aspect as well in case of SDK.

Reply:
Is the SDK open source?

Reply:
Yes the SDK is open source.

**Ref**: [AWS Java SDK](https://github.com/aws/aws-sdk-java)